### DISCUSSION TITLE
Consolidate "requested a deployment" messages in pull requests

### DISCUSSION THREAD

**Discussion by aaroncameron-wk** [community/community#20171] [Pull Requests]
Currently, whenever a deployment is requested in a PR, a message shows up for each individual action that relies on it. For repositories with many actions, this creates a ton of clutter on PRs that seems like it could be easily condensed or collapsed into a single message. The only unique information on each line (the timestamp and run status) is duplicated below in the actual actions window.

Example: https://github.com/Arelle/Arelle/pull/222

**Comment by abelbraaksma** [comment]
Indeed. And I'd like to add that this is extremely worrisome, because it will often make review and other comments invisible. Not sure if this can be turned into a change request somewhere, not sure "community discussions" is the best place for it to be picked up by the GitHub team.

Here's an example. And after 10 or so commits, the whole thread becomes extremely hard to navigate because relevant comments get collapsed:

![image](https://user-images.githubusercontent.com/16015770/211821976-90c8a141-2620-47f1-b78f-55855d5df033.png)

**Comment by abelbraaksma** [comment]
To anyone else wanting to get rid of all these "inactive" comments that add no actual information at all, using this "Refined Github" browser extension (Chrome, Safari, Firefox) makes them disappear instantly: https://github.com/refined-github/refined-github

Of course, ideally GitHub should just collapse these, but that may never happen. Browser extension is the only way afaik.

**Comment by Splines** [comment]
Duplicate of https://github.com/orgs/community/discussions/17666
